dashboards/vmalert.json

```diff
@@ -2565,140 +2569,765 @@
       "type": "row"
     },
     {
-      "collapsed": true,
       "datasource": {
-        "type": "datasource",
-        "uid": "grafana"
+        "type": "prometheus",
```

Comment:
The changes seem too disruptive for one line change. Please consider using the same Grafana version as this dashboard suggests. And make sure you use "Export for sharing externally" option:
<img width="338" alt="image" src="https://github.com/VictoriaMetrics/VictoriaMetrics/assets/2902918/164f76b8-585a-48e2-ba08-ffec572a6cee">

Or update the json manually.